addons/website_slides/models/res_partner.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class ResPartner(models.Model):
+    _inherit = 'res.partner'
+
+    slide_channels_count = fields.Integer('Followed Course Count', compute='_compute_channels_count')
+    company_slide_channels_count = fields.Integer('Followed Course By Company Partners Count', compute='_compute_company_channels_count')
```

Comment:
I would name it slide_channels_company_count so that is is both 'prefixed' with slide_channels

Reply:
+1

And maybe label to Company Course Count ?